src/lib/stores/workflows.ts

```diff
@@ -46,7 +68,13 @@ const updateWorkflows: StartStopNotifier<WorkflowExecution[]> = (set) => {
         }
 
         if (error) {
-          workflowError.set(error);
+          if (hideWorkflowQueryErrors) {
+            workflowError.set(
+              'A error has occurred while querying for Workflows.',
```

Comment:
Do we want to go ahead and add a `translate` here?

Reply:
Yup good call, added it.